### ISSUE TITLE
PlutoReport presentation mode e.g. nextslide, does not work with new Pluto version

### ISSUE THREAD

**Issue by dmetivie** [DhruvaSambrani/PlutoReport.jl#17]
Thanks for nice package!
I tried using notebooks with Pluto `v0.19.27` and PlutoReport, it was working.
Now with Pluto `v0.19.29` the button next slide do not work anymore. 
Maybe it is related to [add keyboard shortcuts (LeftArrow and RightArrow) to move slides in presentation mode](https://github.com/fonsp/Pluto.jl/pull/2611) that was added in `v0.19.28`?

**Comment by DhruvaSambrani** [comment]
Yes, this package has gone out of date from the Pluto version. Ideally, I will need to refix quite a few things.

You are welcome to fix stuff!
